communication/icons/hack/convert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+HEIGHT=249
+WIDTH=256
+
+docker build -t svgconvertor:latest -f hack/svgconvertor/Dockerfile .
```

Comment:
if you put the Dockerfile in this dir, you would not have weird relative path stuff :)